lib/strings.dart

```diff
@@ -19,7 +19,12 @@ This app is designed to be used by Anaesthetic and Critical Care doctors at West
   // *** WARNING ***
   // Increment by one when the disclaimerBody changes. This will force users to accept/re-accept the disclaimer again
   // if they have previously accepted it. If the disclaimer has changed then users need to re-accept it.
-  static const String disclaimerVersion = '1';
+  static const String disclaimerCurrentVersion = '1';
```

Comment:
Actually thinking about it, might be better to make this a number type so can do '>' comparison on it

Reply:
Actually I intentionally made it String. Because its only important that it be different, not incrementing. I think its one of those rare cases where ... handy to be a number because humans relate well to that + only important that we detect a change (which we'll be doing manually).

Reply:
Also we may have to roll back a version of the Disclaimer, which if a number then makes it interesting.

Reply:
Each release code only holds the current version flag, so only important to detect device version flag vs current app version flag. The app can't know previous values in previous app releases. .... I thought about this one a lot before making it a number as String.

Reply:
@maks I can change all the code to make it an int ... but it won't get us anything. behaviour will be exactly the same.